Automattic/wp-calypso
# change the support button placement

some minor fixes in styles

Related to #
2648-gh-tumblr/a8c-dsp

## Proposed Changes

we want to move the support button in the campaign details page on the blaze press pages
![Screenshot 2024-10-07 at 16 32 58](https://github.com/user-attachments/assets/886c00e3-eb50-4f7c-a05c-b39889885ee2)

there are a few more UI fixes that this PR is addressing.. 

- we changed the support links area 
- we moved the blaze powered by jetpack 
- we changed the `automattic/components` button in favor of the  `@wordpress/components`- the former is now deprecated

before: 
![Screenshot 2024-10-07 at 16 34 14](https://github.com/user-attachments/assets/20425443-a090-49ff-8018-0c175c9ed47d)

notice that there is no background color on the box on the above ss

after
![Screenshot 2024-10-07 at 16 34 24](https://github.com/user-attachments/assets/c546981f-a1a6-4350-a4d3-5edfd0d5db8d)
and the powered by jetpack was moved under the main content (left column)
 
![Screenshot 2024-10-07 at 16 35 52](https://github.com/user-attachments/assets/b8d4c5ae-a691-4731-80b0-b84cc7085e97)


- a new function was added to check if the promote again button should be displayed. before we were using the canCancelCampaign. now the promote again can be used in more cases (if the campaign is active etc) 
if the campaign was rejected though the promote again wont be shown (i guess we don't want to make it easy to users to promote a campaign that has been rejected by the moderation, so that is it hard for them to spam the queue)

if the campaign is active both buttons (cancel campaign and promote again) can be shown at the same time.. 

in mobile it looks like this

![Screenshot 2024-10-07 at 16 40 51](https://github.com/user-attachments/assets/99ed0598-ed90-410d-8867-e09ac458d603)

in desktop like this

![Screenshot 2024-10-07 at 16 42 00](https://github.com/user-attachments/assets/ab4fc40a-1f95-4337-af1c-a137b1aede9e)


## Testing Instructions

please checkout the PR (or use the preview link) and check the UI 
ensure that nothing seems broken etc

check in different screensizes to simulate mobile, tablet desktop etc

do a CR

## Pre-merge Checklist
